src/server/index.tsx

```diff
@@ -393,6 +393,13 @@ async function createSsrHtml(root: string, isoData: IsoDataOptionalSite) {
   </head>
 
   <body ${helmet.bodyAttributes.toString()}>
+    <a
+    href="#main-content"
+    class="skip-link"
+    onclick="document.getElementById('main-content').focus(); return false;"
+    >
+      Jump to content
```

Comment:
This is going to need a translation. [Please add one to the translations repo](https://github.com/LemmyNet/lemmy-translations).